src/main/app/src/components/RemoveButton.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 import IconButton from '#/src/components/IconButton';
 
-export default function RemoveButton({ children, ...props }) {
+export default function RemoveButton({ children = undefined, ...props }) {
```

Comment:
Mä ny kirjotan mielipiteitä tänne kun tulee mieleen mutta ei tarvii tehdä jos on eri mieltä 😄 

Refaktoroinnin ja yleisen tiedostohaun tukemisen kannalta on kivempi jos default exportteja pyrkisi välttämään. Itse käytän paljon perus "Find in all files" jolloin voi käydä tällasia mitä aikasemmas projektissa oli ettei löydykään kaikkia käyttökeissejä hakemalla komponentin nimellä:

filuA:
export default KomponenttiX

filuB:
import OlenkinImportattuEriNimelläX from './KomponenttiX'

Ja sitten tekee filuA ison refaktoroinnin ja tajuaa sitten että sitä käytettiinkin jossain muualla ja refaktorointityö onkin suunniteltua isompi. Tää mainittu rivi on ny triviaalikeissi mutta joskus hommat venähti kun joku oli sniikannu uudelleennimeämisen importtiin.

Reply:
Tästä olen kyllä ihan samaa mieltä, mutta ei ole tullut toistaiseksi keskityttyä noihin. Kun/jos nuo korjaa, olisi hyvä laittaa esim tämä päälle: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-default-export.md . Muutenkin mun mielestä olisi hyvä ehkä käyttää eslinttiä ohjaamaan vähän tiukemmin joissakin tapauksissa.